eth_pydantic_types/abi.py

```diff
@@ -0,0 +1,50 @@
+from typing import ClassVar
+
+from eth_pydantic_types.numbers import Int as _Int
+from eth_pydantic_types.numbers import UInt as _UInt
+from eth_pydantic_types.numbers import _make_cls
+from eth_pydantic_types.string import String as _String
+
+
+class Int(_Int):
```

Comment:
I was wondering if we should make all the class names lowercase in the `abi` module to reflect actual Solidity types, sort of like how `eip712` package is doing it.